mylist.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*args)
+    @list = args
+  end
```

Comment:
- [ ] You did an amazing job and your enumerable methods are working very fine and pass all the test cases given. But it seems you missed one project requirement. In MyList you are required to implement a method `each` that yields successive members of `@list`. Kindly make changes. :pray:

Reply:
Hey @henatan99,  thanks for your feedback! I dont quite understand that part. yields successive members... Should I create a method that prints out the list members with list.each method?

Reply:
def each
      @list.each {
        |n| puts n 
      }
    end

Is something like this is what the description requires?

Reply:
Kind of, but better to use `yield` method

Reply:
Ok, nice to make changes even though it is not exactly as I expected. It is not explicitly stated in the requirements hoe the `each` method behave.